### PR TITLE
Spider Infestation Balance Rebalance Expansion Part 1

### PR DESCRIPTION
**Pull request by Hoolny**
## Changelog
:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
sound: added/modified/removed audio or sound effects
image: added/modified/removed some icons or images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by SethLafuente** [comment]
This isnt ready to get merged just published it to show it to a mainteiner

**Comment by Rex9001** [comment]
Uhm seth you forgot to write a pr body

**Comment by Overuseddart** [comment]
Seth please get GDP I want to see this merged.

**Comment by moocowswag** [comment]
looks good and aimed towards something that spiders want to be, an antag that slowly tries to gain ground/control of the station via webs.

But i dont think this will set them far apart from xenos.

xenos currently are very mobile and can instantly just relocate via space due their resistance to lack of atmos + how quickly they can make a new base since theyll have plasma stored up.

spiders are forced to be mobile and relocate because they have no actual way of fixing or combating shitmos, they can only delay it, this combined with the fact that most of them are slow and immobile/hard to relocate + it takes awhile for them to get a new base with webs going means that they suffer so much more from the playstyle that they are often forced into.

**Comment by SethLafuente** [comment]
> looks good and aimed towards something that spiders want to be, an antag that slowly tries to gain ground/control of the station via webs.
> 
> But i dont think this will set them far apart from xenos.
> 
> xenos currently are very mobile and can instantly just relocate via space due their resistance to lack of atmos + how quickly they can make a new base since theyll have plasma stored up.
> 
> spiders are forced to be mobile and relocate because they have no actual way of fixing or combating shitmos, they can only delay it, this combined with the fact that most of them are slow and immobile/hard to relocate + it takes awhile for them to get a new base with webs going means that they suffer so much more from the playstyle that they are often forced into.
> 
> so im worried they will just remain as worse xenos.
> 
> especially on maps like icebox where its warm enough to where a lizard can indefinitely survive while wearing one of the countless winter coats on the station, but spiders will quickly die when encountering the same cold that even a lizard can ignore.

yeah this is often an Issue I hope my addition of web passages and solid webs that increase spider base strength and the increase of broodmothers health to 200 instead of 60 also enriched eggs also spawn as a spiderlings so broodmother spiderlings can leave in vents and relocate to a new base not to mention the new spider types such as scout spider that can detect possible threats to walls move in vents to find good spots to set up new colonies amongst many other things

web passages basically let spiders walk through a structure while atmos cant pass through it and its dense so it also hides whatever is behind it meanwhile solid webs are far stronger than sealed webs and can take 3 times more damage than normal webs

basically all these changes lead up to spiders being able to protect and remain in the territory they have while not letting them just full out kill the entire station through hit and run crew wipe attacks
